docs/starting/configuring.rst

```diff
@@ -101,3 +101,6 @@ behavior.
     e.g. ``logging.WARNING``.
     Defaults to ``None``.
     
+
+``WAFFLE_ADMIN_PAGES_ARE_ENABLED``
+    Admin pages can be disabled via this configuration. It is True by default.
```

Comment:
Enables the default admin pages for Waffle models. This is `True` by default, but can be disabled to override or customize the pages.

Reply:
Done ✅